core-build/gulp-core-build-typescript/src/BaseCmdTask.ts

```diff
@@ -127,6 +127,14 @@ export abstract class BaseCmdTask<TTaskConfig extends IBaseCmdTaskConfig> extend
       }
 
       binaryPackagePath = this.taskConfig.overridePackagePath;
+
+      // try to get the package at the override
+      const newPackageAbsPath: string = path.resolve(this.taskConfig.overridePackagePath);
+      const newPackageJsonPath: string | undefined = BaseCmdTask._getPackageJsonPath(newPackageAbsPath);
+      if (newPackageJsonPath) {
```

Comment:
If this is `undefined`, that probably indicates an issue. Maybe we should print a warning in that case and continue to use the OOB package.

Reply:
Yea that is a great idea.  Let me add that.